libraries/AP_OSD/AP_OSD_Screen.cpp

```diff
@@ -1820,6 +1838,20 @@ void AP_OSD_Screen::draw_current2(uint8_t x, uint8_t y)
     draw_current(1, x, y);
 }
 
+void AP_OSD_Screen::draw_vtx_power(uint8_t x, uint8_t y)
+{
+    AP_VideoTX *vtx = AP_VideoTX::get_singleton();
+    if (!vtx) {
+        return;
+    }
+    uint16_t powr = 0;
+    // If currently in pit mode, just render 0mW to the screen
+    if(!vtx->has_option(AP_VideoTX::VideoOptions::VTX_PITMODE)){
```

Comment:
@andyp1per This is not doing what I expected. When I am in pit mode, the displayed number does not show zero. Instead, it bounces between 1 and <current power setting>.

Reply:
It will not show 0 - we leave the power value alone so that when you exit pitmode you will be back at that power level. I think this is also what the VTX generally returns - pitmode is independent of the "current" power. If you want to show 0 you will need to do that explicitly here.

Reply:
oh sorry, now I understand the question you are asking. Hmmn, you are just querying a parameter here should not be changing - unless you are setting one of the other VTX_OPTIONS?

Reply:
It happened once more, but then it never happened again and now works as expected. I will keep experimenting to see if I can identify what triggered this edge case.

Reply:
A description of what happened:
- My tbs unify pro32 1.1 was in pit mode. But ardupilot did not recognize that it was in pit mode (i.e. the first options bit was 0)
- vtx_get_power_mw() kept returning either 1 or 25. When I set power to 500 then it returned either 1 or 500.
- It's possible that I powered on my flight controller a few seconds before powering my vtx

Reply:
Anyway, I am satisfied that this is unrelated to this PR, so I will go ahead and remove the WIP label.